### DISCUSSION TITLE
How can tart be installed from github releases

### DISCUSSION THREAD

**Discussion by tnxz** [cirruslabs/tart#484] [Q&A] [answered]
Tried installing but ...
<img width="372" alt="Screenshot 2023-04-29 at 8 32 44 AM" src="https://user-images.githubusercontent.com/91027295/235280603-f6b0e616-6126-4a38-a9dc-59fa65b59926.png">

**Comment by fkorotkov** [comment]
Please either install Tart binary via Homebrew:

```bash
brew install cirruslabs/cli/tart
```

Or you can always download an archive with latest binary via this [link](https://github.com/cirruslabs/tart/releases/latest/download/tart.tar.gz).

**Reply by tnxz** [reply]
forgot to mention the issue with tart binary
```
$ ./tart --help

zsh: killed     ./tart --help
```

**Reply by fkorotkov** [reply, marked as answer]
If you install from the archive, you need to invoke `tart.app/Contents/MacOS/tart` binary so `tart.app/Contents/embedded.provisionprofile` are picked up and you can use Bridged network.
